feature/mypage/src/main/java/com/napzak/market/mypage/component/MyPageMenuCard.kt

```diff
@@ -0,0 +1,116 @@
+package com.napzak.market.mypage.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import com.napzak.market.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.mypage.MyPageMenu
+import com.napzak.market.util.android.noRippleClickable
+
+private const val ROW_COUNT = 2
+private const val COLUMN_COUNT = 3
+
+@Composable
+fun MyPageMenuCard(
+    onSalesClick: () -> Unit,
+    onPurchaseClick: () -> Unit,
+    onRecentClick: () -> Unit,
+    onFavoriteClick: () -> Unit,
+    onSettingsClick: () -> Unit,
+    onHelpClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+
+    val menus = listOf(
+        MyPageMenu.SALES to onSalesClick,
+        MyPageMenu.PURCHASE to onPurchaseClick,
+        MyPageMenu.RECENT to onRecentClick,
+        MyPageMenu.FAVORITE to onFavoriteClick,
+        MyPageMenu.SETTINGS to onSettingsClick,
+        MyPageMenu.HELP to onHelpClick,
+    )
```

Comment:
p1: 리컴포지션이 발생했을 때 불필요한 객체 생성이 발생할 수 있어서 remember로 감싸주면 좋을 것 같아요!
```suggestion
    val menus = remember{
          listOf(
             MyPageMenu.SALES to onSalesClick,
             MyPageMenu.PURCHASE to onPurchaseClick,
             MyPageMenu.RECENT to onRecentClick,
             MyPageMenu.FAVORITE to onFavoriteClick,
             MyPageMenu.SETTINGS to onSettingsClick,
             MyPageMenu.HELP to onHelpClick,
        )
    }
```